### ISSUE TITLE
Add examples in filter editor

### ISSUE THREAD

**Issue by laurenwalker** [NCEAS/metacatui#1867]
### Ideally
We could have a "Fill in an example" button that fills out the filter editor with a coherent example, so users get an idea of what  to do. I've seen this done in UIs as a "randomize" button.

### Faster implementation
If we get feedback from users about needing examples, I think an example screenshot or two would work well as a stepping stone to our ideal version.

**Comment by laijasmine** [comment]
Will we be able configure a custom example for a specific repositories? I've found that suggesting researchers use the NSF award number for the ADC related portals often provides a good starting point for our users.

**Comment by laurenwalker** [comment]
Yes, we can design this so that repositories can use their own custom examples
